internal/pkg/source/source.go

```diff
@@ -226,7 +229,7 @@ func (s *Source) visit(n ssa.Node, maxInstrReached map[*ssa.BasicBlock]int, last
 		s.visitOperands(n, maxInstrReached, lastBlockVisited)
 
 	// These nodes are both Instructions and Values, and currently have no special restrictions.
-	case *ssa.Field, *ssa.MakeInterface, *ssa.Select, *ssa.TypeAssert:
+	case *ssa.Field, *ssa.MakeInterface, *ssa.Select, *ssa.Slice, *ssa.TypeAssert, *ssa.UnOp:
```

Comment:
These are both needed for the tests in this PR.

Reply:
This might be a good indication that the PR has gotten a bit bigger in scope than intended.

Reply:
Poking at the test cases more, I see now that this is necessary for propagating in `taint(&myString, someSource)`.

Reply:
Hmm. That's not what I'm seeing. Should we consolidate this discussion with the discussion in this thread: https://github.com/google/go-flow-levee/pull/185/files#r525629809